pkgs/development/python-modules/imbalanced-learn/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchPypi, scikitlearn, pandas, nose, pytest }:
+
+buildPythonPackage rec {
+
+    pname = "imbalanced-learn";
+    version = "0.3.3";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "1r5js9kw6rvmfvxxkfjlcxv5xn5h19qvg7d41byilxwq9kd515g4";
+    };
+
+    preConfigure = ''
+      export HOME=$PWD
```

Comment:
Is this needed for anything other than the tests? If not, it belongs into `checkPhase`.

Reply:
Tests only. moved to checkPhase.